apps/i18n/common/en_us.json

```diff
@@ -2384,6 +2384,7 @@
   "sectionsNotificationJoinFull": "We couldn't add you to section {sectionId} because there are already {sectionCapacity} students in it. Please contact your teacher for help.",
   "sectionsNotificationJoinNotFound": "Section {sectionId} doesn't exist.",
   "sectionsNotificationJoinSuccess": "You joined section {sectionName}.",
+  "sectionsNotificationJoinSuccessForNonPl": "You joined section {sectionName}. This is not a Professional Learning section so it will be visible on the [Teacher Dashboard]({teacherDashboardUrl}).",
```

Comment:
@dmcavoy is "Teacher Dashboard" a phrase we have shown to teachers in the past? Or is there a better name for that page?

Reply:
I think we call this the Teacher Homepage in general.

Reply:
I'll update that!